src/providers/boleto-api-caixa/index.js

```diff
@@ -120,10 +120,14 @@ const translateResponseCode = (axiosResponse) => {
   const hasErrors = path(['errors', 'length'], axiosResponseData)
 
   if (!hasErrors) {
+    const boletoResponseLinks = path(['links'], axiosResponseData)
+    const boletoUrl = boletoResponseLinks[0].href
```

Comment:
Tem possibilidade de `boletoResponseLinks` não voltar um array ou não ter pelo menos 1 item? Só falando pq se sim, vai quebrar a linha 124

Reply:
Então, uma vez que volte sem erros, sempre vai ter os links para acessar o boleto sim. Vc acha que vale a pena uma validação msm assim?

Reply:
Pessoalmente sim, mas vai na tua 🤘 😬 

Se por algum acaso nao vir link, ou vir num formato nao esperado, vai gerar erro tentando acessar uma propriedade que nao existe `href`, mas se tiver segura que sempre vem, de boa 😄

Reply:
Uma possível sugestão (totalmente opcional kkkk) também é usar o próprio path para evitar essa chance de quebrar por campo undefined

```suggestion
    const boletoUrl =  path(['links', '0', 'href'], axiosResponseData)
```

Reply:
ó, seguinteeeee
Olhei na docs da boleto-api e quando o boleto é registradom, ctz que vem com link! Mas eles não falam lá a ordem que vem, pq pode ser que venha link de pdf tb, dai o que fiz foi dar um jeito de pegar o link do HTML.
Usei mto ramda kkkkk aceito sugestões se acharem outro jeito melhor